docker-compose.yml

```diff
@@ -136,7 +136,7 @@ services:
   msa-ui:
     restart: unless-stopped
     container_name: msa_ui
-    image: openmsa/openmsa:msa2-ui-2.8.9-659e3a5e5c2b729c900ae21882eafda7ee1b0a44
+    image: ubiqube/msa2-ui:d30496c46ff944230b860ec027b37e7698131948
```

Comment:
@speUbiqube @antoine-brun , why local images ?

Reply:
this is the fix for https://jiras.ubiqube.com/browse/MSA-13192